app/models/rejection.rb

```diff
@@ -11,4 +11,12 @@ class Rejection < ActiveRecord::Base
   belongs_to :visit
 
   validates :reason, inclusion: { in: REASONS }
+
+  def pvo_possible?
```

Comment:
@kyliemulholland “manage all of that stuff through NOMIS”… for now! Dun-dun daaa